index.src.html

```diff
@@ -3279,6 +3289,14 @@
 : <dfn attribute for=VideoFrame>[[color space]]</dfn>
 :: The {{VideoColorSpace}} associated with this frame.
 
+: <dfn attribute for=VideoFrame>\[[metadata]]</dfn>
```

Comment:
Is this `\` correct here?

Reply:
Yes, otherwise, we do not get link. This is done for duration for instance.